impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -154,12 +154,12 @@ struct TexImage2DData {
         // only use the stencil component.
         //
         // https://registry.khronos.org/OpenGL-Refpages/gl4/html/glTexImage2D.xhtml
+      case PixelFormat::kD24UnormS8Uint:
```

Comment:
hmm, should we have been hitting this path before?

Reply:
I don't think we ever hit this path if offscreen MSAA is supported.